src/webui/src/App.tsx

```diff
@@ -30,7 +30,9 @@ export const AppContext = React.createContext({
     // eslint-disable-next-line @typescript-eslint/no-empty-function, @typescript-eslint/no-unused-vars
     changeMetricGraphMode: (val: 'max' | 'min') => {},
     // eslint-disable-next-line @typescript-eslint/no-empty-function, @typescript-eslint/no-unused-vars
-    changeEntries: (val: string) => {}
+    changeEntries: (val: string) => {},
```

Comment:
I think `(_: string) => {}` will suppress the warning.

Why `no-empty-function` is enabled? Doesn't make any sense to me.

Reply:
It makes sense to me. And this rule is discussed result?